src/loaders/lottie/jerryscript/jerry-core/jrt/jrt.h

```diff
@@ -23,8 +23,13 @@
 #include "jerryscript-port.h"
 #include "jrt-types.h"
 
-
-#define JERRY_NDEBUG
+#ifndef _MSC_VER
+  #define JERRY_NDEBUG
+#else
+  #ifdef NDEBUG
```

Comment:
instead of doing this, use the option `b_ndebug` of the meson project, see https://mesonbuild.com/Builtin-options.html#base-options